theme/static/js/create-resource.js

```diff
@@ -97,6 +97,10 @@ $(document).ready(function () {
                 }
             });
 
+            $(".btn-cancel-create-resource").click(function () {
+                history.back();
```

Comment:
Let's make this explicitly go to the my resources page via `window.location.` `history.back` can sometimes be unpredictable.

Reply:
"Create Resource page" might be invoked from another location when design change in the future. Using "history.back" can prevent we revisit the code again.

Reply:
Ok, fair point. Ship it